plugins/USBqemu/qemu-usb/usb-ohci.cpp

```diff
@@ -356,7 +356,7 @@ static void ohci_copy_td(OHCIState *ohci, struct ohci_td *td,
 
     ptr = td->cbp;
     n = 0x1000 - (ptr & 0xfff);
-    if (n > len)
+    if (n > static_cast<uint32_t>(len))
```

Comment:
Can len be negative ? Maybe it could be better to change the parameter of the function instead ? (disclaimer I barely read the diff)